src/lib.rs

```diff
@@ -607,18 +607,15 @@ impl Config {
                     cmd.arg("-AARM64");
                 } else if target.contains("i686") {
                     use cc::windows_registry::{find_vs_version, VsVers};
-                    match find_vs_version() {
```

Comment:
Can unrelated changes like this be split out into separate PRs and/or separate commits?

Reply:
Removed the lint changes